tox.ini

```diff
@@ -4,7 +4,7 @@ skipsdist = True
 
 [testenv]
 usedevelop = True
-commands = py.test {posargs:tests/} --assert=plain
```

Comment:
Would `tox -e py35 -- some/test/path/test_foo.py` still work with this change? We might need to keep `{posargs}`.